source/beamline/BioXAS/BioXASSideM2Mirror.cpp

```diff
@@ -0,0 +1,18 @@
+#include "BioXASSideM2Mirror.h"
+
+BioXASSideM2Mirror::BioXASSideM2Mirror(QObject *parent) :
+    BioXASM2Mirror(parent)
+{
+	// Initialize member variables.
+
+	screen_ = new AMPVControl("BioXAS Side M2 Mirror Screen Control", "VSC1607-5-I22-02:InBeam", "VSC1607-5-I22-02:InBeam", QString(), this);
```

Comment:
You could use AMSinglePVControl for succinctness.  It's purely aesthetic in that it is a convenience constructor for AMPVControl when there is only one PV for moving, feedback, and has no stop.